python/python/res/job_queue/ext_job.py

```diff
@@ -143,17 +143,19 @@ def get_max_running_minutes(self):
     def set_max_running_minutes(self, min_value):
         self._set_max_running_minutes(min_value)
 
-    def minimumArgumentCount(self):
+    @property
+    def min_arg(self):
         return self._min_arg()
-
-    def maximumArgumentCount(self):
+    
+    @property
+    def max_arg(self):
         return self._max_arg()
 
-    def argumentTypes(self):
+    def get_argument_types(self):
```

Comment:
Well;

if we are on the property bandwagon I suggest:
   
@property
def arg_types(self):
     ...